prime-router/src/main/resources/metadata/hl7_mapping/resources/ServiceRequest/OBR.yml

```diff
@@ -29,21 +29,6 @@ elements:
     value: [ '%resource.occurrence.extension(%`rsext-hl7v2-date-time`).value' ]
     hl7Spec: [ '%{obrFieldPath}-6' ]
 
-  - name: observation-collected-datetime
```

Comment:
These mappings were moved out to both ServiceRequest/OBRExtension and Specimen/OBR for a few different reasons:
- The mappings you see here look at `%resource.specimen` to find the values. ORU is not currently populating ServiceRequest.specimen. (This might be incorrect according to the mapping inventory). Moving these mappings out allow ORU and OML to each access these fields differently.
- ORM does not map to a Specimen resource at all, thus to preserve the fields they need to be in OBRExtension.

Reply:
This might be incorrect according to the mapping inventory - can we ticket this?